src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -130,6 +131,11 @@ export class CommentEditorComponent implements OnInit {
       return;
     }
 
+    if (this.uploadService.isVideoFile(filename) && file.size >= MAX_VIDEO_UPLOAD_SIZE) {
+      this.handleUploadError('Oops, video is too big. Keep it under 2MB.', insertedText);
```

Comment:
For standardisation I think we can extract the error message to `VIDEO_SIZE_EXCEEDED_ERROR` under `upload.service`, similar to the one below!